src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -109,8 +109,8 @@ def ensure_bicep_installation(release_tag=None, target_platform=None, stdout=Tru
                 print(f"Installing Bicep CLI {release_tag}...")
             else:
                 print("Installing Bicep CLI...")
-        ca_file = certifi.where()
-        request = urlopen(_get_bicep_download_url(system, release_tag, target_platform=target_platform), cafile=ca_file)
+        os.environ.setdefault("CURL_CA_BUNDLE", certifi.where())
+        request = urlopen(_get_bicep_download_url(system, release_tag, target_platform=target_platform))
```

Comment:
Copying @emitor's comment from https://github.com/Azure/azure-cli/commit/e0bb745bdc15b4c958baaddc1357aa78e7a3d94e#r71858253:

Hi @wwmoraes, `urlopen` does not use the OS environment `CURL_CA_BUNDLE` as `request` does. So if you remove the `cafile=ca_file` parameter this break the command `az bicep` if you are behind a proxy using a custom CA. 

I'm trying to using this at work behind a corporate proxt and does not work anymore and throws the `[SSL: CERTIFICATE_VERIFY_FAILED]`. I've manually modified the file locally and is working.

In order to do it properly so `urlopen` does not throw any warnings, you should change it to something like this:
``` python
...
import ssl
...
        context = ssl.create_default_context(cafile=certifi.where())
        request = urlopen(_get_bicep_download_url(system, release_tag, target_platform=target_platform), context=context)
...
```

Would you be able to fix this back please?

Thanks!

Reply:
In short, `urlopen` doesn't support `REQUESTS_CA_BUNDLE` or `CURL_CA_BUNDLE` and we shouldn't make `urlopen` support them either.